pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y

```

Comment:
Should be 'return x - y' rather than 'return y - y'

Reply:
New version should have the requested changes